vale.ini

```
@@ -36,3 +36,17 @@ Canonical.025b-latinisms-to-reconsider = suggest
 Canonical.025c-latinisms-to-avoid = warning
 Canonical.400-Enforce-inclusive-terms = error
 Canonical.500-Repeated-words = NO
+
+[*.rst]
+BasedOnStyles = Canonical
+
+# relevant block directives
+BlockIgnores = (?m)^(\s*)\.\.\s+(code|code-block|sourcecode|literalinclude)::.*$(\n(?:\1\s+.*)?)+
+
+# inline literal roles
+TokenIgnores = :relatedlinks:, ``.+?``, :samp:`.+?`, :file:`.+?`, :command:`.+?`
+TokenIgnores = :program:`.+?`, :literal:`.+?`, :kbd:`.+?`, :file:`.+?`, :math:`.+?`
+TokenIgnores = :token:`.+?`, :regexp:`.+?`, :command:`.+?`, :option:`.+?`, :envvar:`.+?`
+
+# links defined elsewhere
+TokenIgnores = `\w+`_
```

```suggestion
TokenIgnores = \`\w+\`_
```

Testing with this token as it was caused issues. This fix seems to work for the syntax we are targeting without falsely allowing negative cases (e.g. the existing test000 case).